test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -55,9 +56,25 @@
 (def script-to-check-if-constant-is-defined
   "! $instance_id.nil?")
 
+(defn await-jruby-agents
+  "Wait for pending actions on agents associated with the jruby pool
+  service to free up."
+  [{:keys [pool-agent flush-instance-agent]}]
+  ;; Dump a dummy action into each of the jruby pool agents and await that
+  ;; action's completion.  This assumes that any previously existing
+  ;; actions would have been flushed through prior to the new one being
+  ;; processed.
+  (jruby-agents/send-agent pool-agent #(constantly true))
```

Comment:
My reading of the documentation for await makes it seem like these dummy actions wouldn't be needed to ensure that all previous actions have been flushed through

```
Blocks the current thread (indefinitely!) until all actions dispatched thus far,
from this thread or agent, to the agent(s) have occurred.
```
and also like maybe you could just do `(await pool-agent flush-instance-agent)`, but maybe I'm missing something?